### ISSUE TITLE
Escaped underscore in a doi field

### ISSUE THREAD

**Issue by GeoBosh** [iridia-ulb/references#8]
This is one of the entries in #7. I believe that the underscore in the `doi` field should not be escaped (not sure, this may depend on the bibtex style).
```
@incollection{CarPinOli2017recipe,
  ...
  crossref =     "EUROGP2017",
  doi =          {10.1007/978-3-319-55696-3\_16},
}
```

**Comment by MLopez-Ibanez** [comment]
Fixed by commit 713d5aa93e9f041d43c5cd1239da3871a8577ca8

The working DOI can be tested here: https://iridia-ulb.github.io/references/#CarPinOli2017recipe

DOIs are URLs, thus if a bibstyle cannot handle it, the style needs to be fixed or the DOI command defined by the style redefined.

**Comment by MLopez-Ibanez** [comment]
And thanks for the report! Very useful!
